x-pack/plugins/index_management/server/routes/api/component_templates/register_component_template_routes.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { RouteDependencies } from '../../../types';
+
+import { registerGetAllRoute } from './get';
+import { registerCreateRoute } from './create';
+import { registerUpdateRoute } from './update';
+import { registerDeleteRoute } from './delete';
+
+export function registerComponentTemplateRoutes(dependencies: RouteDependencies) {
```

Comment:
WDYT of moving all of this code into `index.ts`? This way there's roughly a 1-to-1 relationship with the `register_<thing>.ts` files and the routes that get registered. I did this in ILM if you want to see how it would look.